### ISSUE TITLE
Brace-less listeners incorrectly report syntax error

### ISSUE THREAD

**Issue by TapGhoul** [kizeevov/slint-idea-plugin#91]
For any listener, if you have a callback without {} it shows as a syntax error, despite being valid.

For example,

```
export component Example {
    callback hello;

    TouchArea {
        clicked => root.hello();
    }
}
```
This compiles fine, and reports no issues in slintpad, but reports as problematic in intellij.
